test_data/rules/use_decorated_box.dart

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) 2021, the Dart project authors. Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+// test w/ `dart test -N use_decorated_box`
```

Comment:
We should also have a test case in which a `Container` is created with a `decoration` and at least one other argument other than `key` or `child`.